src/main/java/uk/gov/hmcts/juror/job/execution/jobs/contentstore/ContentStoreFileJob.java

```diff
@@ -29,7 +30,7 @@ public abstract class ContentStoreFileJob extends LinearJob {
 
     private static final String UPDATE_SQL_QUERY = "UPDATE CONTENT_STORE "
         + "SET DATE_SENT=current_date "
-        + "WHERE REQUEST_ID=? AND FILE_TYPE=?";
+        + "WHERE DOCUMENT_ID=? AND FILE_TYPE=?";
```

Comment:
Is DOCUMENT_ID always unique?

Reply:
The DOCUMENT_ID is unique.  It is generated by the stored procedure and includes the REQUEST_ID:

'JURY'||LPAD(**v_revision**::VARCHAR(22),4,'0')||'01.0001',

<img width="571" alt="image" src="https://github.com/hmcts/juror-scheduler-execution/assets/148471056/48c847e1-8a97-4165-9657-4d4d33ccdb97">

<img width="321" alt="image" src="https://github.com/hmcts/juror-scheduler-execution/assets/148471056/90eaf82b-9a8c-4db7-90c1-271ff8483261">